### ISSUE TITLE
[Bug]: Build UBUNTU >23.4 does not work

### ISSUE THREAD

**Issue by vdijken** [Sigil-Ebook/Sigil#768]
### Bug Description

1. Library qt6-5compat-dev does not exist
2. Build failes

### Platform (OS)

Linux

### OS Version / Specifics

15.0-118-generic #128-Ubuntu SMP Fri Jul 5 09:28:59 UTC 2024 x86_64 x86_64 x86_64 GNU/Linux

### What version of Sigil are you using?

2.2.0 (Flatpak)

### Any backtraces or crash reports

```shell
-- CMake version in use: 3.29.0
-- Using older findpython cmake module
-- Using Bundled ZLIB
-- Using Bundled MiniZip
-- Using Bundled Hunspell
-- Using Bundled PCRE2
CMake Error at src/qt6sigil.cmake:31 (find_package):
  Could not find a configuration file for package "Qt6" that is compatible
  with requested version "6.4".

  The following configuration files were considered but not accepted:

    /usr/lib/x86_64-linux-gnu/cmake/Qt6/Qt6Config.cmake, version: 6.2.4
    /lib/x86_64-linux-gnu/cmake/Qt6/Qt6Config.cmake, version: 6.2.4

Call Stack (most recent call first):
  src/CMakeLists.txt:659 (include)
```

**Comment by kevinhendricks** [comment]
We do not make nor endorse the flatpak builds.  A third party handles that.

Newer versions of Sigil (Sigil 2.3.0 is now out) no longer require Qt6-compat5 to build, but they do require Qt6 version 6.4.0 or later to build.

If you build things yourself, you can always use the python module "atqinstall" to grab a local version of Qt 6.7.2 for your platform and build against it yourself.

See this thread in our MobileRead Sigil User forum for more detailed instructions:

https://www.mobileread.com/forums/showthread.php?t=362308&highlight=aqtinstall&page=3

Starting with post #33 or so.

**Comment by kevinhendricks** [comment]
The best place to get help building Sigil is in our Sigil Forum on www.mobileread.com (See our README.md).  Not here in issues.  So closing this.  Please ask for help in our User Forum.

**Comment by dougmassay** [comment]
2.2.0 should build with Qt6.2.4. we didn't bump the minimum to Qt6.4 until Sigil 2.3.0. But regardless... qt6-5compat-dev package is available from Ubuntu 23.10 (Mantic) on. As is Qt6.4. Something is messed up. You're indicating Sigil 2.2.0, but the build log shows it's clearly looking for  Qt6.4--which didn't become the Qt minimum until Sigil 2.3.0.

Need much more clarification to help. As Kevin mentioned, take this to our Mobileread user forums as this not really a bug.
